spikeinterface/widgets/unitwaveforms.py

```diff
@@ -139,6 +139,7 @@ def _do_plot(self):
                 if self._plot_waveforms and self._plot_templates:
                     color = 'k'
                 ax.plot(xvectors_flat, template.T.flatten(), lw=1, color=color)
+                ax.set_title('template %s' %unit_ids[i])
```

Comment:
Use `'f{}'` style formatting.

Reply:
Ok, will do

Reply:
I think I will refactor BaseWIdget and MultiWidget into the same object to get something more convinient in any case.
When ready you will be able to rebase on top of it.